tagstore/backends/django.py

```diff
@@ -24,52 +24,64 @@ def create_taxonomy(self, name: str, owner_id: Optional[UserId]) -> TaxonomyMeta
                 external_id=owner_id.external_id,
             )
         obj = TaxonomyModel.objects.create(name=name, owner=owner_obj)
-        return TaxonomyMetadata(uid=obj.id, name=name, owner_id=owner_id)
+        return Taxonomy(uid=obj.id, name=name, owner_id=owner_id, tagstore=self)
 
-    def get_taxonomy(self, uid: int) -> Optional[TaxonomyMetadata]:
+    def get_taxonomy(self, taxonomy_uid: TaxonomyId) -> Optional[Taxonomy]:
         try:
-            tax = TaxonomyModel.objects.get(pk=uid)
+            tax = TaxonomyModel.objects.get(pk=taxonomy_uid)
         except TaxonomyModel.DoesNotExist:
             return None
-        return tax.as_tuple
+        return tax.as_tuple(self)
 
-    def _add_tag_to_taxonomy(self, taxonomy_uid: int, tag: str, parent_tag: Optional[str] = None) -> str:
+    def _add_tag_to_taxonomy(self, taxonomy_uid: TaxonomyId, name: str, parent_tag: Optional[str] = None) -> str:
         if parent_tag:
             # Check the parent tag:
             try:
-                pt = TagModel.objects.get(taxonomy_id=taxonomy_uid, tag=parent_tag)
+                pt = TagModel.objects.get(taxonomy_id=taxonomy_uid, name=parent_tag)
             except TagModel.DoesNotExist:
                 raise ValueError("Invalid parent tag.")
```

Comment:
If you provide tests for these two lines, [41-42](https://github.com/open-craft/blockstore/pull/15/files#diff-b4fdc37c88d35346e1890617408283a8R41), and [141-142](https://github.com/open-craft/blockstore/pull/15/files#diff-b4fdc37c88d35346e1890617408283a8R141) below, then your coverage for the `tagstore` dir will reach 100%.